frontend/public/components/overview.jsx

```diff
@@ -2,14 +2,16 @@ import * as React from 'react';
 import * as _ from 'lodash-es';
 import * as fuzzy from 'fuzzysearch';
 import * as PropTypes from 'prop-types';
-import { default as Toolbar } from 'patternfly-react/dist/esm/components/Toolbar/Toolbar';
+import * as classnames from 'classnames';
+
```

Comment:
nit: remove the space... we typically group all imports from yarn dependencies together